src/He.PipelineAssessment.Data/ExtendedSinglePipeline/EsriSinglePipelineExtendedClient.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace He.PipelineAssessment.Data.ExtendedSinglePipeline
+{
+    public interface IEsriSinglePipelineExtendedClient
+    {
+        Task<string?> GetSinglePipelineExtendedData(string spid);
+        Task<SinglePipelineExtendedDataList?> GetSinglePipelineExtendedDataList(int offset);
+    }
+    public class EsriSinglePipelineExtendedClient : IEsriSinglePipelineExtendedClient
+    {
+
+        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly ILogger<EsriSinglePipelineExtendedClient> _logger;
+        private readonly IEsriSinglePipelineExtendedDataJsonHelper _jsonHelper;
+
+        public EsriSinglePipelineExtendedClient(IHttpClientFactory httpClientFactory, ILogger<EsriSinglePipelineExtendedClient> logger, IEsriSinglePipelineExtendedDataJsonHelper jsonHelper)
+        {
+            _httpClientFactory = httpClientFactory;
+            _logger = logger;
+            _jsonHelper = jsonHelper;
+        }
+
+        public async Task<string?> GetSinglePipelineExtendedData(string spid)
+        {
+            string? data = null;
+            string whereClause = $"sp_id={spid}";
+            string outFields = "*";
+
+            var relativeUri = $"query?where={whereClause}&outFields={outFields}&f=json";//potentially needs a token on the end also
+
+            using (var response = await _httpClientFactory.CreateClient(ClientConstants.SinglePipelineExtendedClient)
+                       .GetAsync(relativeUri)
+                       .ConfigureAwait(false))
+            {
+                data = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                if (!response.IsSuccessStatusCode)
+                {
+                    _logger.LogError($"StatusCode='{response.StatusCode}'," +
+                                     $"\n Message= '{data}'," +
+                                     $"\n Url='{relativeUri}'");
+
+                    return null;
+                }
+            }
+
+            return data;
+        }      
+
+        public async Task<SinglePipelineExtendedDataList?> GetSinglePipelineExtendedDataList(int offset)
+        {
+            string? data = null;
+            string whereClause =
+                "sp_status in ('Active', 'Approved','In Programme') AND sp_stage in ('Lead', 'Opportunity', 'Contracted','Holding') AND sp_business_area in ('Investment', 'Development', 'MPP') " +
+                "AND (project_owner is NOT NULL AND project_owner <> '') " +
+                "AND (units_or_homes is NOT NULL) " +
+                "AND (local_authority is NOT NULL AND local_authority <> '') " +
+                "AND (funding_ask is NOT NULL) " +
+                "AND (pipeline_opportunity_site_name is NOT NULL AND pipeline_opportunity_site_name <> '') " +
+                "AND (applicant_1 is NOT NULL AND applicant_1 <> '') " +
+                "AND land_type in ('Greenfield', 'Brownfield', 'Mixed')";
```

Comment:
have you tested this locally? as just checked the extended data source and for example it doesn't have the land_type field
just wondering if we ever need this method at all, as not sure if we are planning to be syncing from the extended data source? probably not harm leaving it, but the query needs testing